src/core/server/rendering/rendering_service.test.ts

```diff
@@ -127,4 +140,35 @@ describe('RenderingService', () => {
       });
     });
   });
+  describe('checkURLvalid()', () => {
+    it('URL is valid', async () => {
+      jest.mock('axios', () => ({
+        async get() {
+          return {
+            status: 200,
+          };
+        },
+      }));
+      const result = await service.checkURLValid(
+        'https://as2.ftcdn.net/v2/jpg/02/23/11/43/500_F_223114340_8AdZa15EeRHbMVQmbPEuzJutqHpEXIeV.jpg',
```

Comment:
if you update the default to be opensearch_dashboards_logo, then this should be the opensearch url since probably don't random real links in the repo